xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-ui/src/main/resources/XWiki/Ratings.xml

```diff
@@ -394,9 +394,10 @@ position: relative;
     <className>XWiki.UIExtensionClass</className>
     <guid>d2420ff0-a5b9-4fae-a798-fc623aac7a6b</guid>
     <property>
-      <content>{{include reference='XWiki.RatingsMacros'/}}
-
-{{velocity}}
+      <content>{{velocity}}
+#if (!$isGuest)
```

Comment:
My use case: guest user with view rights. So this might be a mix of being guest or not having view rights. WDYT ?